app/views/submissions/_assigned_evaluators.html.erb

```diff
@@ -0,0 +1,46 @@
+<div class="usa-accordion">
+  <% rand = SecureRandom.hex(8) %>
+  <h2 class="usa-accordion__heading" id="<%= "#{rand}-accordion-heading" %>">
+    <button
+      type="button"
+      class="usa-accordion__button display-flex flex-justify icon-white"
+      aria-expanded="true"
+      aria-controls="<%= rand %>"
+    >
+      <div>Assigned Evaluators</div>
+    </button>
+  </h2>
+
+  <div class="usa-accordion__content" id="<%= rand %>">
+      <table class="usa-table usa-table--stacked-header usa-table--borderless gray-header">
+          <thead>
+            <tr>
+              <th scope="col">Evaluator name</th>
+              <th scope="col">Evaluation Status</th>
+              <th scope="col">Score</th>
+              <th scope="col" aria-label="Actions"></th>
+            </tr>
+          </thead>
+          <tbody>
+            <% @submission.evaluators.each do |evaluator| %>
```

Comment:
we should filter this by evaluator_submission_assignments with assigned or recused status. (this should work, or you can assign the filtered list in the controller like you do with @available_evaluators)
```suggestion
            <% @submission.evaluators.where("evaluator_submission_assignments.status" => ["assigned", "recused"]).each do |evaluator| %>
```